agents/build_agent.py

```diff
@@ -35,6 +35,7 @@ def get_instructions() -> str:
       just return the error message. Otherwise, start with `builder-live.log` and try to identify
       the build failure. If not found, try the same with `root.log`. Summarize the findings
       and return them as `error`. If the build failed due to a build timeout, set `is_timeout` to `true` in your output.
+      If the build succeeded, return the URL to the built package in `url`.
```

Comment:
This makes no sense. First, URLs to built artifacts are already returned from the `build_package` tool, second, there could be multiple RPMs (even hundreds in some cases) - which one should the model choose? The correct answer is all of them, but that's not always possible, they could conflict with each other for example. It would be best to leave this to Testing Farm and pass a Copr build ID instead of a RPM.